content/en/contribution/_index.md

```diff
@@ -44,7 +44,7 @@ section2:
           link: ''
 
   organization:
-    topic: 'Community Organization'
+    topic: 'Community Framework'
```

Comment:
`Framework` is not that exact. I think `Governance` is much better.

Reply:
I think the original word "Organization" here means to tell how the community is structured while it should be Framework here, but Governance is ok here